src/main/java/org/openjdk/jextract/impl/ClassSourceBuilder.java

```diff
@@ -144,27 +142,27 @@ void appendIndentedLines(String s) {
     }
 
     final void emitDefaultConstructor() {
-        appendIndentedLines(STR."""
+        appendIndentedLines(String.format("""
```

Comment:
Suggestion - tweak the `appendIndentedLines` to call `String::format` inside that method, so that all clients don't have to call `String::format` manually from outside.

Reply:
Would this work? 

Before
```
 appendIndentedLines(String.format("""
                private static final MethodHandle %s = %s.sliceHandle(%s);
                """,
                arrayHandleName, fieldLayoutName, path));
```
After 
```
void appendIndentedLines(String s, String... args) {
        sb.appendIndentedLines(String.format(s,args));
    }
```
and the call would look like 
```
appendIndentedLines("private static final MethodHandle %s = %s.sliceHandle(%s);"
                                , arrayHandleName, fieldLayoutName, path));
```
probably would create a new method rather than remove the old one